README-eebus.md

```diff
@@ -0,0 +1,18 @@
+# Installation
```

Comment:
können wir die zusätzlichen readmes in den docs Ordner verschieben? Als eebus.md?